src/main/java/uk/gov/hmcts/reform/blobrouter/services/EnvelopeService.java

```diff
@@ -55,17 +58,10 @@ public UUID createRejectedEnvelope(
     ) {
         eventRecordRepository.insert(new NewEventRecord(containerName, blobName, Event.REJECTED, rejectionReason));
 
-        return createEnvelope(containerName, blobName, blobCreationDate, Status.REJECTED);
-    }
-
-    private UUID createEnvelope(String containerName, String blobName, Instant blobCreationDate, Status status) {
-        return envelopeRepository.insert(new NewEnvelope(
-            containerName,
-            blobName,
-            blobCreationDate,
-            now(),
-            status
-        ));
+        return envelopeRepository
```

Comment:
should we also create event corresponding to this envelope or will that be introduced separately ?

Reply:
it is already created one line above